core/src/nucleus/ribosome/api/init_globals.rs

```diff
@@ -22,7 +22,7 @@ pub fn invoke_init_globals(
     _args: &RuntimeArgs,
 ) -> Result<Option<RuntimeValue>, Trap> {
     let globals = InitGlobalsOutput {
-        app_name: runtime.app_name.to_string(),
+        app_name: runtime.dna_name.to_string(),
```

Comment:
I kinda agree with this, but then it's also confusing cause on the left hand "app_name" and on the right name "dna_name" and it makes you wonder about the difference, or how/why they're the same, and one is set to the other

Reply:
Yes. They are the same, and I'll rename all mention of 'app' in hdk-rust in a subsequent PR.
Seems out-of-scope of this one.
Expect: `APP_GLOBALS.app_name` => `ZOME_API_GLOBALS.dna_name`

Reply:
Cool. I did. It's a good change to make prior to the dev preview release